sample_apps/run_example.sh

```diff
@@ -3078,6 +3194,52 @@ function run_test_simple_app_under_app_service() {
     run_simple_app_as_client_talk_to_Cert_Service
 }
 
+# ###########################################################################
+# Driver method to run simple_app steps with different pairs of public-key
+# and symmetric-key crypto algorithms. This interface is mainly a
+# testing interface to validate that different pairs of crypto algorithms
+# work correctly.
+# ###########################################################################
+function run_test-crypto_algorithms() {
+
+    run_cmd
+
+    start_certifier_service
+
+    subcase=0
+    # These valid-lists are found in certifier_algorithms.c
+    # public-key algo 'ecc-384' is failing ... Comment out for now ...
+    # for public_key_algo in rsa-3072 rsa-4096 ecc-384; do
+    for public_key_algo in rsa-3072 rsa-4096; do
```

Comment:
Attn: @jlmucb - We need to discuss and reconcile exactly what public-key algorithms are supported.

I thought I could list many from `certifier_algorithms.cc` as they are enumerated in the newly-added `Enc_public_key_algorithms[]` array.

But due to the checks in `src/cc_helpers.cc/generate_auth_key()`, only a few public-key algorithms are actually supported or can be used in this pair-wise algorithms testing scheme.

For instance, an attempt to use `ecc-384`, i.e., `Enc_method_ecc_384`, will fail due to the check here in the above mentioned routine:

```
1154 bool certifier::framework::cc_trust_manager::generate_auth_key(bool regen) {
[...]
1188   } else {
1189     printf("%s() error, line %d, Unsupported public key algorithm: '%s'\n",
1190            __func__,
1191            __LINE__,
1192            public_key_algorithm_.c_str());
1193     return false;
1194   }
```